src/Directory.Build.props

```diff
@@ -0,0 +1,50 @@
+<Project>
+  <PropertyGroup>
+    <VersionPrefix>1.9.9</VersionPrefix>
```

Comment:
I'm a bit confused to why the version is at two places now?

Reply:
By default, MSBUILD doesn't merge Directory.Build.props. Fixed